src/integration-tests/introspector/introspectTest.sh

```diff
@@ -393,6 +418,65 @@ function deployCreateDomainJobPod() {
   waitForPod $pod_name
 }
 
+#
+# Create the model in image docker image
+#
+function createMII_Image() {
+  trace "Info: Create MII Image"
+
+  mkdir -p ${test_home}/mii/workdir/models || exit 1
+  cp ${SCRIPTPATH}/mii/models/*  ${test_home}/mii/workdir/models || exit 1
+  cd ${test_home}/mii/workdir  || exit 1
+  echo "place holder" > dummy.txt || exit 1
+  zip ${test_home}/mii/workdir/models/archive.zip dummy.txt > /dev/null 2>&1 || exit 1
+
+  (
+  export WORKDIR=${test_home}/mii/workdir  || exit 1
+  export MODEL_IMAGE_TAG=it || exit 1
+  export MODEL_IMAGE_NAME=model-in-image || exit 1
+  export MODEL_IMAGE_BUILD="when-missing"
+
+  docker rmi ${MODEL_IMAGE_NAME}:${MODEL_IMAGE_TAG} --force > /dev/null 2>&1
+
+  trace "Info: Downloading WDT and WIT..."
+
+  ${SOURCEPATH}/kubernetes/samples/scripts/create-weblogic-domain/model-in-image/build_download.sh \
+   > ${test_home}/miibuild_download.out 2>&1
+
+  if [ $? -ne 0 ] ; then
+    trace "Error: createMII_Image: download tools failed"
+    cat ${test_home}/miibuild_download.out
+    exit 1
+  fi
+
+  trace "Info: Launching WIT to build the image..."
+
+  ${SOURCEPATH}/kubernetes/samples/scripts/create-weblogic-domain/model-in-image/build_image_model.sh \
+   > ${test_home}/miibuild_image.out  2>&1
+
+  if [ $? -ne 0 ] ; then
+    trace "Error: createMII_Image: build image failed"
+    cat ${test_home}/miibuild_image.out
+    exit 1
+  fi
+  )
+  export WEBLOGIC_IMAGE_NAME=model-in-image || exit 1
+  export WEBLOGIC_IMAGE_TAG=it || exit 1
+
+  kubectl -n $NAMESPACE delete configmap ${DOMAIN_UID}-wdt-config-map --ignore-not-found || exit 1
+  kubectl -n $NAMESPACE create configmap  ${DOMAIN_UID}-wdt-config-map \
+        --from-file=${SCRIPTPATH}/mii/wdtconfigmap | tracePipe "Info: kubectl output: "
+
+  kubectl -n $NAMESPACE label  configmap ${DOMAIN_UID}-wdt-config-map  weblogic.domainUID=$DOMAIN_UID || exit 1
+
+  kubectl -n $NAMESPACE delete secret ${DOMAIN_UID}-runtime-encryption-secret --ignore-not-found || exit 1
+  kubectl -n $NAMESPACE create secret generic  ${DOMAIN_UID}-runtime-encryption-secret \
+        --from-literal=password=welcome1 | tracePipe "Info: kubectl output: "
+
+  kubectl -n $NAMESPACE label secret ${DOMAIN_UID}-runtime-encryption-secret weblogic.domainUID=$DOMAIN_UID || exit 1
+
+}
+
 #############################################################################
 #
 # Run introspection "JobPod", parse its output to files, and put files in a cm
```

Comment:
Please change ")" to ") || exit 1" at line 646 - this is an old bug unrelated to your current change. Otherwise, if the yaml template substitution fails, then the script will keep on truckin' and fail later rather than at the point of the original failure...

e.g. change:

```
  ( # Generate svc yaml from template
    export SERVER_NAME="${server_name}"
    export SERVICE_INTERNAL_PORT="${internal_port}"
    export SERVICE_EXTERNAL_PORT="${external_port}"
    export SERVICE_NAME=${service_name}
    ${SCRIPTPATH}/util_subst.sh -g wl-nodeport-svc.yamlt ${target_yaml} || exit 1
  )

```

to 

```
  ( # Generate svc yaml from template
    export SERVER_NAME="${server_name}"
    export SERVICE_INTERNAL_PORT="${internal_port}"
    export SERVICE_EXTERNAL_PORT="${external_port}"
    export SERVICE_NAME=${service_name}
    ${SCRIPTPATH}/util_subst.sh -g wl-nodeport-svc.yamlt ${target_yaml} || exit 1
  ) || exit 1
```

Reply:
Thanks for doing that.